jetty-servlet/src/main/java/org/eclipse/jetty/servlet/BaseHolder.java

```diff
@@ -185,6 +186,38 @@ public synchronized boolean isInstance()
         return _instance != null;
     }
 
+    protected <W> T wrap(final T component, final Class<W> wrapperFunctionType, final BiFunction<W, T, T> function)
```

Comment:
I don't love this signature.... but I'd like it a lot more with javadoc!

Reply:
Added javadoc.